src/main/java/com/maxi/calendar/model/User.java

```diff
@@ -0,0 +1,42 @@
+package com.maxi.calendar.model;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name="user")
+@AllArgsConstructor @Getter @Setter
+public class User {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private int id;
+	
+	private String name;
+	private String role;
+	
+	@ManyToMany(mappedBy="users")
+	private List<Event> events = new ArrayList<>();
+	
+	public User() {
+		
+	}
+	
+	public User(String name, String role) {
```

Comment:
you should create a Factory or Builder pattern for this object creation

Reply:
I fixed putting @builder lombock annotation